packages/spl-utils/src/mplAssetAPI.ts

```diff
@@ -164,3 +163,42 @@ export async function getAssetsByOwner(
   }
 }
 
+export type SearchAssetsOpts = {
+  sortBy?: { sortBy: "created"; sortDirection: "asc" | "desc" };
+  page?: number;
+};
+
+export async function searchAssets(
```

Comment:
is this what you were thinking @ChewingGlass ?

Reply:
I would allow people to pass the params directly. Given they may be different for each call.